### PR TITLE
Functionality for the today bool.

### PR DESCRIPTION
**Pull request by millidavids**

### CODE REVIEW

File: server.go [hunk 1]
```diff
@@ -59,6 +65,50 @@ func hacker_handler(w http.ResponseWriter, r *http.Request) {
...
+
+  var gh []GitHubEvent
+
+  answer := false
+
+  now_time := time.Now()
+
+  now_string := now_time.String()
+
+  today_string := now_string[0:10]
```
Comment (chaserx) [inline server.go]: what's this magic number?

Comment (millidavids) [inline server.go]: grabs "2015-01-10"

Comment (millidavids) [inline server.go]: I'll probably try regex next to match the date to the time.Now().String().

**Comment by chaserx** [comment]
cool. so all you have left is to push those github lookup funcs into the create or show actions, right?

**Comment by millidavids** [comment]
Not sure. I haven't really looked into the actions at all for golang yet. Is the show action not the json response? Because I call the today func and set the TodayUTC variable as the that json is displayed.

**Comment by millidavids** [comment]
I was kind of hoping to get this merged and then work on the CRUD stuff in another branch.

**Comment by chaserx** [comment]
Yeah. So we have two "controller actions" already: show and create. You'd just need your github activity json parsing functions to tie into one or both of these.

**Comment by millidavids** [comment]
Yeah, it does. Assuming that the github name is valid, each time the show action is hit, the today func is called and sets the TodayUTC variable to true or false depending on whether or not the user had any events that day.

**Comment by chaserx** [comment]
+1

**Pull request merged**